services/server/src/lib/session.ts

```diff
@@ -63,19 +64,18 @@ export function canBook({
 }
 
 // todo: write tests
-export async function canJoinCall({
+export async function canJoinSession({
   userId,
-  callId,
-  callType,
+  sessionId,
 }: {
   userId: number;
-  callId: number;
-  callType: ICall.Type;
+  sessionId: string;
 }) {
   const now = dayjs.utc();
+  const sessionType = sessionId.split(":")[0];
```

Comment:
For convenience and readability we can use `first` from lodash. 
```suggestion
  const sessionType = first(sessionId.split(":"));
```